### PR TITLE
Ignored RHEL6.x for ostree tests

### LINKED ISSUE #3638: RHEL6 os_tree tests not been skipped
Test: https://satellite-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/satellite6-automation-downstream-rhel6-tier1/104/testReport/tests.foreman.ui.test_repository/RepositoryTestCase/test_positive_create_custom_ostree_repo/

Decorator: https://github.com/SatelliteQE/robottelo/pull/3629/files#diff-4ac5ea985efa38f0c89842de24c052e6R525

### PR DESCRIPTION
**Pull request by renzon**
close #3638
Improved skip_if_os decorator logging to easier debug
Improved test_host tests to fail instead of skipping
when unittest2.SkipTest is raised

### CODE REVIEW

File: tests/robottelo/decorators/test_host.py [hunk 1]
```diff
@@ -177,21 +184,45 @@ class SkipIfOSTestCase(SkipIfOSIsUnavailableTestCase):
...
+
+        @host.skip_if_os('RHEL7.1')
+        def dummy():
+            return True
+
+        self.assertRaises(unittest2.SkipTest, dummy)
+
+    def test_skipping_with_single_major_version(self):
+        """Test skipping when decorator params is equals to host version but
+        omits minor and patch"""
```
Comment (oshtaier) [inline tests/robottelo/decorators/test_host.py]:
```
"""Test skipping when decorator params is equals to host version but
omits minor and patch
"""
```

File: tests/robottelo/decorators/test_host.py [hunk 2]
```diff
@@ -177,21 +184,45 @@ class SkipIfOSTestCase(SkipIfOSIsUnavailableTestCase):
...
+
```
Comment (oshtaier) [inline tests/robottelo/decorators/test_host.py]: redundant line

File: tests/robottelo/decorators/test_host.py [hunk 3]
```diff
@@ -177,21 +184,45 @@ class SkipIfOSTestCase(SkipIfOSIsUnavailableTestCase):
...
         def dummy():
             return True
 
         self.assertRaises(unittest2.SkipTest, dummy)
 
+    def test_skipping_with_single_minor_version(self):
+        """Test skipping when decorator params is equals to host version but
+        omits patch
+        """
+
```
Comment (oshtaier) [inline tests/robottelo/decorators/test_host.py]: redundant line

File: robottelo/decorators/host.py [hunk 4]
```diff
@@ -66,24 +68,29 @@ def wrapper(*args, **kwargs):
...
-                    'Skipping test %s in module %s due to %s',
-                    func.__name__,
-                    func.__module__,
-                    skip_msg
-                )
+                skip_template = 'Skipping test %s in module %s due to %s'
+                log_version_info(skip_msg, skip_template)
                 raise unittest2.SkipTest(skip_msg)
             else:
+                debug_msg = 'host {0} not in ignored versions {1}'.format(
```
Comment (oshtaier) [inline robottelo/decorators/host.py]: Seems a little bit redundant as here we gonna add same 2000 debug messages

**Comment by oshtaier** [comment]
ACK

**Comment by rochacbruno** [comment]
ACK

**Pull request merged**